packages/react-core/src/components/Label/Label.tsx

```diff
@@ -18,6 +23,8 @@ export interface LabelProps extends React.HTMLProps<HTMLSpanElement> {
   color?: 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'orangered' | 'grey' | 'gold';
   /** Variant of the label. */
   variant?: 'outline' | 'filled' | 'overflow' | 'add';
+  /** Status of the label with a respective icon and color. */
```

Comment:
Could we just add a small disclaimer that this will override the `color` prop (maybe add something to icon that it will override the default status icon as well).

Reply:
Agree

Reply:
Added to the `status` and `icon` prop descriptions.